Library/Homebrew/test/support/lib/utils/ruby.rb

```diff
@@ -0,0 +1,12 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  def self.ruby_exec_args
```

Comment:
Feels a little weird to override the existing method here.

Reply:
It's what we already do for `config.rb`. We insert `HOMEBREW_LIBRARY_PATH/test/support/lib` into the `$LOAD_PATH` and that directory has a replacement for config.rb which replaces all the prefixes etc. This is the same mechanism as that, with the goal of allowing that `$LOAD_PATH` we inserted be passed on to further forks.

I can't think of any way cleaner that doesn't start bleeding test logic into actual code, such as special ENV variables.

Reply:
Could this just live in `config.rb` somehow (or make it a constant there or something)? I think it just feels weird to overload a `utils` method.

Reply:
So basically make this array a constant that lives there? I could do that.

Reply:
Aye, that'd be nice 👍🏻